rotkehlchen/chain/ethereum/constants.py

```diff
@@ -11,3 +12,34 @@
 ZERO_ADDRESS = string_to_ethereum_address('0x0000000000000000000000000000000000000000')
 
 CPT_KRAKEN = 'kraken'
+
+WEIGHTED_ETHEREUM_NODES = (
```

Comment:
If any of them fail all the time we can (1) temporarily remove them and (2) contact them about it.

Reply:
we should still contact the ones that fail often imo

Reply:
Okey, will send email to both of them